recipe/meta.yaml

```diff
@@ -1,44 +1,40 @@
-{% set version = "0.20.3" %}
+{% set version = "0.21.0" %}
 
 package:
   name: cartopy
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/C/Cartopy/Cartopy-{{ version }}.tar.gz
-  sha256: 0d60fa2e2fbd77c4d1f6b1f9d3b588966147f07c1b179d2d34570ac1e1b49006
+  sha256: ce1d3a28a132e94c89ac33769a50f81f65634ab2bd40556317e15bd6cad1ce42
 
 build:
   error_overlinking: true
   skip: true  # [py<38]
-  number: 2
+  number: 0
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - cython                                 # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
-    - proj                                   # [build_platform != target_platform]
     - geos                                   # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
     - python
     - pip
-    - setuptools-scm
+    - setuptools-scm >=7.0.0
     - cython
     - numpy
-    - proj
     - geos
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    - shapely >=1.6.4
+    - shapely >=1.6.4,<2
     - pyshp >=2.1
-    - six >=1.3.0
-    - proj
-    - pyproj >=3.0
+    - pyproj >=3.0.0
```

Comment:
The release states that `proj` is no longer a direct dependency and is only accessed through `pyproj`.